dialect_ydb.go

```diff
@@ -0,0 +1,126 @@
+//go:build !no_ydb
+// +build !no_ydb
+
+package goose
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/ydb-platform/ydb-go-sdk/v3"
+	"github.com/ydb-platform/ydb-go-sdk/v3/table"
+	"github.com/ydb-platform/ydb-go-sdk/v3/table/types"
+)
+
+func init() {
```

Comment:
Hmm, this PR introduces a few new concepts (7? new interfaces) in order to add support for ydb?

This leads to additional overhead and I'm not sure the trade-off is worth it for us.

Can this be implemented in a way that uses existing primitives?

Reply:
Can you get me advice, how to implement named parameters? Ydb support named parameters only.